src/fides/cli/commands/util.py

```diff
@@ -130,8 +130,8 @@ def up(ctx: click.Context, no_pull: bool = False, no_init: bool = False) -> None
     Starts the sample project via docker compose.
     """
 
+    echo_green("Checking Docker is running and version is compatible...")
```

Comment:
This shouldn't be green:
```suggestion
    click.echo("Checking Docker is running and version is compatible...")
```

Reply:
I prefer a green message- moving this back to where it was